.github/workflows/run-tests.yml

```diff
@@ -6,19 +6,7 @@ name: Run Tests
 on: [push, pull_request]
 
 jobs:
-  tests:
-    runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [14.x, 16.x]
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm run build
-    - run: npm test
+  call_run_tests:
+    uses: yext/slapshot-reusable-workflows/.github/workflows/run_tests.yml@v1
```

Comment:
`v1` is referring to a branch name in this case. the reusable workflows repo has a `v1` branch that's based off of main in case we want to add more versions later